core/src/main/kotlin/no/strooped/view/screen/components/PlacementLabel.kt

```diff
@@ -0,0 +1,54 @@
+package no.strooped.view.screen.components
+
+import com.badlogic.gdx.Gdx
+import com.badlogic.gdx.graphics.Color
+import com.badlogic.gdx.graphics.g2d.BitmapFont
+import com.badlogic.gdx.math.Vector2
+import com.badlogic.gdx.scenes.scene2d.ui.Label
+
+class PlacementLabel(
+    val placement: Int,
+    position: Vector2,
+    width: Float,
+    font: Float
+) : Label("", LabelStyle(BitmapFont(Gdx.files.internal("applegothic.fnt")), Color.WHITE)) {
+    init {
+        var textToDisplay = placement.toString()
+        @Suppress("MagicNumber") run {
```

Comment:
These lines do a lot of different things and should instead be placed inside one or multiple functions

Reply:
done